Tests/PostgresNIOTests/New/Extensions/PSQLBackendMessageEncoder.swift

```diff
@@ -188,19 +188,10 @@ extension PSQLBackendMessage.BackendKeyData: PSQLMessagePayloadEncodable {
     }
 }
 
-extension PSQLBackendMessage.DataRow: PSQLMessagePayloadEncodable {
+extension DataRow: PSQLMessagePayloadEncodable {
     public func encode(into buffer: inout ByteBuffer) {
-        buffer.writeInteger(Int16(self.columns.count))
-        
-        for column in self.columns {
-            switch column {
-            case .none:
-                buffer.writeInteger(-1, as: Int32.self)
-            case .some(var writable):
-                buffer.writeInteger(Int32(writable.readableBytes))
-                buffer.writeBuffer(&writable)
-            }
-        }
+        buffer.writeInteger(Int16(self.columnCount))
```

Comment:
`self.columnCount` is already an `Int16`, so if you want to ensure being explicit about the type, why not use `.writeInteger(self.columnCount, as: Int16.self)` instead of ostensibly invoking `Int16.init<Int16>(_:)` (though the compiler hopefully inlines that away)?